scripts/casefold_db.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python
+# Copyright 2021 The Matrix.org Foundation C.I.C.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import sqlite3
+from typing import Any, Dict, List, Tuple
+from tests.utils import ResolvingMemoryReactorClock
+
+import signedjson.sign
+
+from sydent.util import json_decoder
+from sydent.util.emailutils import sendEmail
+from sydent.sydent import Sydent, parse_config_file, get_config_file_path, parse_config_dict
+from sydent.util.hash import sha256_and_url_safe_base64
+
+def calculate_lookup_hash(sydent, address):
+        cur = sydent.db.cursor()
+        pepper_result = cur.execute("SELECT lookup_pepper from hashing_metadata")
+        pepper = pepper_result.fetchone()[0]
+        combo = "%s %s %s" % (address, "email", pepper)
+        lookup_hash = sha256_and_url_safe_base64(combo)
+        return lookup_hash
```

Comment:
this looks like it's indented too much (or with tabs or something)

Think this file needs linting -- please can you add it to the lint script (assuming we have one in Sydent)? :)